Rules/UseConsistentIndentation.cs

```diff
@@ -123,6 +150,27 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                         AddViolation(token, indentationLevel++, diagnosticRecords, ref onNewLine);
                         break;
 
+                    case TokenKind.Pipe:
+                        var pipelineIsFollowedByNewlineOrLineContinuation = k < tokens.Length - 1 && k > 0 &&
+                              (tokens[k + 1].Kind == TokenKind.NewLine || tokens[k + 1].Kind == TokenKind.LineContinuation);
+                        if (!pipelineIsFollowedByNewlineOrLineContinuation)
+                        {
+                            break;
+                        }
+                        if (pipelineIndentationStyle == PipelineIndentationStyle.IncreaseIndentationAfterEveryPipeline)
+                        {
+                            AddViolation(token, indentationLevel++, diagnosticRecords, ref onNewLine);
+                        }
+                        else if (pipelineIndentationStyle == PipelineIndentationStyle.IncreaseIndentationForFirstPipeline)
```

Comment:
Can use `break` above and eliminate `else` here

Reply:
Although this would probably be more natural as a switch

Reply:
I used the proposed break statement in the first `if` statement to convert the `else if` to an `if` statement in [this](https://github.com/PowerShell/PSScriptAnalyzer/pull/1102/commits/bcf7c9ad463d09d0f6f34eb2a8f6954350b9f1cf) commit. A switch statement for 2 cases (out of 3 enum values) is not worth the horizontal space IMHO and does not feel more readable to me (as a reader, I expect a switch statement to go through all enum values)

Reply:
> Shall I merge now?

LGTM